CHANGELOG.md

```diff
@@ -8,6 +8,10 @@
 
 ### App Center Crashes
 
+#### Windows
+
+* **[Fix]** Fix infinite loop when old logs cannot be purged by a new one with a different channel name in a case when the storage is full.
```

Comment:
I think it should be in the `AppCenter` section

Reply:
hmm, but it is only windows related.. It doesn't affect Xamarin